src/roswire/ros2/state.py

```diff
@@ -14,6 +14,47 @@
     from .. import AppInstance
 
 
+@attr.s(frozen=True, auto_attribs=True, slots=True)
+class ROS2SystemState(SystemState):
```

Comment:
For the moment, this is the same as ROS1SystemState, which was the same as the old SystemState. Once this refactoring is pushed, I will start adding the ROS2 specific information here.

Reply:
Would it make more sense to `raise UnimplementedError`?

Reply:
Sorry, I was confused. The implementation is the same as ROS1 _for now_, but is duplicated here as it is going to change in a subsequent PR.